generate_api_json.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 # Relies on the user having cloned the playwright repository in the directory directly adjacent to this repo
 ./clean_generated_files.sh
-mkdir bin; /bin/true
+mkdir bin; true
 pushd ../playwright
 git pull
 node utils/doclint/generateApiJson.js > ../playwright-perl/api.json
 popd
 cp playwright_server bin/playwright_server
 API="$(<api.json)"
-sed -i -e '/%REPLACEME%/r api.json' -e 's/%REPLACEME%//g' bin/playwright_server
+sed -i '' -e '/%REPLACEME%/r api.json' -e 's/%REPLACEME%//g' bin/playwright_server
```

Comment:
IIRC there's no need to supply a blank suffix for backing up the file -- does OSX sed explode on this?

Reply:
Yeah, this breaks linux.  According to the OSX manpage for sed, the old behavior should be what is preferred.  I am going to back this particular patch out.

Reply:
67467610fd69677f0d836361b785773f286eea86